dev/build-system-overrides.nix

```diff
@@ -10,10 +10,6 @@ let
   # `setuptools-scm[toml]` in pyproject.toml would be written as
   # `foo.setuptools-scm = [ "toml" ]` in Nix
   buildSystemOverrides = {
-    arpeggio = {
-      setuptools = [ ];
-      pytest-runner = [ ];
-    };
```

Comment:
Two notes:

1. I moved this from here to `dev/patching-deps.nix` to keep that example self-contained.
2. While moving it, I noticed that the `pytest-runner` dependency seems to be unnecessary, so I removed it.